src/tracer/MOM_tracer_registry.F90

```diff
@@ -848,16 +848,21 @@ end subroutine tracer_Reg_chkinv
 
 
 !> Find a tracer in the tracer registry by name.
-subroutine tracer_name_lookup(Reg, tr_ptr, name)
+subroutine tracer_name_lookup(Reg, n, tr_ptr, name)
   type(tracer_registry_type), pointer    :: Reg     !< pointer to tracer registry
   type(tracer_type), pointer             :: tr_ptr  !< target or pointer to the tracer array
   character(len=32), intent(in)          :: name    !< tracer name
+  integer                                :: n       !< index to tracer registery
```

Comment:
There should be an `intent(out)` declaration  for the argument `n` to `tracer_name_lookup()`.

Reply:
Thank you for adding the intent declaration.